space2stats_api/src/space2stats/main.py

```diff
@@ -0,0 +1,139 @@
+
+
+
+from typing import Dict, List, Literal, Optional
+
+import psycopg as pg
+from psycopg import Connection
+from pydantic import BaseModel
+from geojson_pydantic import Feature, Polygon
+from typing_extensions import TypeAlias
+
+from .h3_utils import generate_h3_ids, generate_h3_geometries
+from .settings import Settings
+
+settings = Settings()
+
+AoiModel: TypeAlias = Feature[Polygon, Dict]
+
+class SummaryRequest(BaseModel):
+    aoi: AoiModel
+    spatial_join_method: Literal["touches", "centroid", "within"]
+    fields: List[str]
+    geometry: Optional[Literal["polygon", "point"]] = None
+
+
+def _get_summaries(fields: List[str], h3_ids: List[str], conn: Connection):
```

Comment:
`private` method with `_` prefix

This is not meant to be used directly by users